client/containers/Booking/Booking.jsx

```diff
@@ -61,44 +58,42 @@ export class Booking extends Component {
                 {
                     this.props.appointment.isSubmitted ?
                         <div className="requestDiv">
-                            <p className="requestText">Your Request has been submitted successfully for Approval</p>
+                            <p className="requestText">You will receive an email once your request is reviewed</p>
```

Comment:
Shouldn't this be like this (take a look at the email I've sent previously)
Your request has been submitted successfully for approval.
You will receive an email once your request is reviewed.